jiant/tasks/tasks.py

```diff
@@ -1545,15 +1548,25 @@ class RTESuperGLUETask(RTETask):
     def load_data(self):
         """ Process the datasets located at path. """
         targ_map = {"not_entailment": 0, True: 1, False: 0, "entailment": 1}
-        self.train_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "train.jsonl"), self._tokenizer_name, self.max_seq_len, targ_map
-        )
-        self.val_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "val.jsonl"), self._tokenizer_name, self.max_seq_len, targ_map
-        )
-        self.test_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "test.jsonl"), self._tokenizer_name, self.max_seq_len, targ_map
-        )
+        def _load_jsonl(data_file):
```

Comment:
can you revert this change? This will fail in that winogender_metrics requires pair_id field.

Reply:
Why should winogender metrics fail on this RTE example?